ui/app/components/das/recommendation-card.js

```diff
@@ -237,8 +247,12 @@ export default class DasRecommendationCardComponent extends Component {
   }
 
   @action
-  recommendationCardDestroying(element) {
-    this.cardHeight = element.clientHeight;
+  cardInserted(element) {
+    this.element = element;
+  }
```

Comment:
We do something like this in multiple places. It might be worth looking into something like [`ember-ref-bucket`](https://github.com/lifeart/ember-ref-bucket) at some point.

Reply:
agreed! This whole thing is more clunky than I’d like 😞